### ISSUE TITLE
Regarding UMAPINFO's ending intermission text

### ISSUE THREAD

**Issue by Mabadman** [coelckers/prboom-plus#536]
Not sure if this is a feature request or a bug report but probably the former.
All testing done on the dev build.

Currently, enabling any ending key (endgame, endpic, endbunny, endcast) will display a short intermission text saying "The End" before the ending screen in shown. Shown here with endcast:
![case0_small](https://user-images.githubusercontent.com/42493726/193033590-0cb8d090-324b-4cb4-9112-191c5282f8ee.gif)

It seems it's not possible to get rid of this intermission text even though it would be preferable for a lot of cases.
Setting `intertext = clear` will skip the ending graphic entirely instead of just skipping the intermission text. Additionally, the game will load the next map in the IWAD instead of loading the map as determined by UM's `next =`. This latter situation is probably not a big deal since the loaded PWAD would have ended by this point so setting a next map would be redundant.
Shown here with `endcast = true` `intertext = clear` and `next = MAP03`:
![case1_small](https://user-images.githubusercontent.com/42493726/193034426-bfe17e10-5d65-40c2-97cc-2a62c56a6259.gif)

Finally, once the entire intermission text is shown it seems holding "use" for longer than a fraction of a second will skip the ending graphic entirely. Once again I'm not sure if this is intended, but it doesn't seem like a preferable situation.
![case2](https://user-images.githubusercontent.com/42493726/193035180-c9b9c4a1-9b90-4687-aa16-c5ae6284be82.gif)

**Comment by fabiangreffrath** [comment]
Regarding the intermission screen, you'll want this:
https://github.com/coelckers/prboom-plus/blob/master/prboom2/doc/umapinfo.md#nointermission

**Comment by Mabadman** [comment]
> Regarding the intermission screen, you'll want this: https://github.com/coelckers/prboom-plus/blob/master/prboom2/doc/umapinfo.md#nointermission
Unfortunately that just gets rid of the intermission screen with the stats, but the text is still there.
![case3](https://user-images.githubusercontent.com/42493726/193056740-f1c04796-7a93-4fa8-9732-bf21f14c84f2.gif)

**Comment by fabiangreffrath** [comment]
> Unfortunately that just gets rid of the intermission screen with the stats, but the text is still there.

And if you explicitly clear that as well?

**Comment by rfomin** [comment]
Yes, there should always be intertext here, this is an implementation detail in PrBoom+, and this affects demo compatibility. Perhaps we can get rid of it in a hypothetical next version of the UMAPINFO specification.

**Comment by Mabadman** [comment]
> > Unfortunately that just gets rid of the intermission screen with the stats, but the text is still there.
> 
> And if you explicitly clear that as well?

Sorry, I was wrong before. I had tested with DSDA Doom.
On PrBoom+'s dev build, setting `nointermission = true` will skip both the intermission text and the stats screen, directly taking the user to the end graphic.
This is certainly a solution to getting rid of the intermission text, but it's also a compromise as in a lot cases it could be preferable to still have have the stats screen show up.

**Comment by NightFright2k19** [comment]
This needs to be fixed, also in Woof.

A code like this 

```
MAP MAP30
{
  intertext = clear
}
```
Is simply supposed to skip the text for the intermission at the end of the game, but instead it shows the level stats, but then no endcast and instead jumps back to MAP01. Regardless what you try, endcast = true, endpic = "CREDIT" or whatever.
